e2e/Makefile

```diff
@@ -2,19 +2,21 @@
 TEST_KUBERNETES_TARGET ?= current
 
 ifeq ($(TEST_KUBERNETES_TARGET),current)
-KUBERNETES_VERSION := 1.21.2
+KUBERNETES_VERSION := 1.22.4
 else ifeq ($(TEST_KUBERNETES_TARGET),prev)
-KUBERNETES_VERSION := 1.20.7
+KUBERNETES_VERSION := 1.21.2
 else ifeq ($(TEST_KUBERNETES_TARGET),prev2)
-KUBERNETES_VERSION := 1.19.11
+KUBERNETES_VERSION := 1.20.7
 endif
 export KUBERNETES_VERSION
 
-KUBE_PROMETHEUS_VERSION := 0.8.0
-TOPOLVM_VERSION := 3f3854c808df5d68f46ba316e96bfe6bb7edcd48
+KUBE_PROMETHEUS_VERSION := 0.9.0
+TOPOLVM_VERSION := 7c269d04b0b8ac8569c8c76c69569a8f9a0b92a8
```

Comment:
7c269d04b0b8ac8569c8c76c69569a8f9a0b92a8  -> https://github.com/topolvm/topolvm/tree/v0.10.3

Reply:
How about simply specifying `v0.13.0` instead of commit hash?